### DISCUSSION TITLE
It is not possible to set a fixed list of wallets

### DISCUSSION THREAD

**Discussion by dwadawdwadwadaw** [WalletConnect/walletconnect-monorepo#4411] [Developer Support]
I have a DApp that uses web3modal and I want the wallet to connect to only 1 - metamask. I tried adding it to featureWalletIds, includeWalletIds and removed BackPack, Coinbase, adding them to excludeWalletIds (since they are displayed now with metamask).

Are there any other options to install only MetaMask in the list or fix it?

`
createWeb3Modal({ 
    wagmiConfig, 
    projectId, 
    defaultChain: bsc, 
    featuredWalletIds: ['c57ca95b47569778a828d19178114f4db188b89b763c899ba0be274e97267d96'],
    includeWalletIds: ['c57ca95b47569778a828d19178114f4db188b89b763c899ba0be274e97267d96'],
    excludeWalletIds: [
        'fd20dc426fb37566d803205b19bbc1d4096b248ac04548e3cfb6b3a38bd033aa',
        '2bd8c14e035c2d48f184aaa168559e86b0e3433228d3c4075900a221785019b0'
    ],
    allowUnsupportedChain: false,
    allWallets: 'HIDE'
})
 `

![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/117940582/e42eac65-c374-494a-b187-06fd229c4397)

**Comment by glitch-txs** [comment]
"WalletConnect" and "Browser Wallet" can't be removed at the moment

**Reply by honeymaro** [reply]
What about removing Coinbase?

**Reply by aminhdev** [reply]
> What about removing Coinbase?


There is an option "enableCoinbase", you can check example below
```
createWeb3Modal({
  wagmiConfig: defaultWagmiConfig({
    ...otherConfig,
    enableCoinbase: false
  }), 
  ...otherConfig
})
```

**Reply by honeymaro** [reply]
Oh, I didn't know that. thanks for your answer. 👍
